### DISCUSSION TITLE
Supabase Image Issue

### DISCUSSION THREAD

**Discussion by KFCNKR** [supabase/supabase#18783] [Questions]
Hi There,

We worked with a developer who used supabase for our website's images. It was a contract-based job. As we finished working together, we did delete his Gmail account.

Recently (after 6-7 months) our website's images seem to be deleted/removed/cannot be reached. We did create his email and sign in with GitHub but as it seems, all of the things, contributions etc. he did -as well as supabase- are gone. Our images cannot be uploaded but the source is from Supabase. How can I fix it.

Your help and support is much appreciated.

**Comment by GaryAustin1** [comment]
You should contact support.
